cmake/D2D_MPI.cmake

```diff
@@ -1,6 +1,12 @@
 # MPI CMakeLists
 
-find_package(MPI REQUIRED)
+find_package(MPI REQUIRED COMPONENTS Fortran)
+
+# adios2 IO backend requires C and C++ MPI components
+if (IO_BACKEND MATCHES "adios2")
+    find_package(MPI REQUIRED COMPONENTS C CXX)
```

Comment:
Should Fortran not also be a required component?

Reply:
It is just on the line above. Don’t know if we want to make all 3 required components independently of ADIOS

Reply:
Ah, missed that!